_config.php

```diff
@@ -12,7 +12,7 @@
  */
 if (class_exists('Resque_Event') && class_exists('SSResqueRun')) {
     Resque_Event::listen('beforeFork', function ($data) {
-        $databaseConfig = DB::getConfig();
+        $databaseConfig = DB::getConfig(DB::CONN_PRIMARY);
```

Comment:
We'll need to bump the framework dependency to use that constant.
In that case we can remove the method check from `connectToDB` as well.

Reply:
Done